docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -0,0 +1,371 @@
+
+## ZSS informational messages
+
+### ZWES0013I
+
+  ZSS Server has started. Version '%s'
+
+  **Reason:**
+
+  ZSS Server has started.
+
+  **Action:**
+
+  No action required.
+
+### ZWES0014I
+
+  ZIS status - '%s' (name='%.16s', cmsRC='%d', description='%s', clientVersion='%d')
+
+  **Reason:**
+
+  The message shows status of the connection to Privileged Server: ZIS status - `<OK or Failure>` (name=`<Privileged Server Name>`, cmsRC=`<RC>`, description=`<description`, clientVersion=`<version>`)
+
+  **Action:**
+
+  if Status is `OK` then no action required. If Status is `Failure` see [ZLUX unable to communicate with zssServer](app-troubleshoot/#zss-server-unable-to-communicate-with-x-mem).
+### ZWES0035I
+
+  ZSS Server settings: Address='%s', port='%d', protocol='%s'
+
+  **Reason:**
+
+  Server is starting using Address=`<IP address>`, port=`<port>`, protocol=`http` or `https`
+
+  **Action:**
+
+  No action required.
+### ZWES0039I
+
+  Installing '%s' service...
+
+  **Reason:**
+
+  `<Service>` is about to install. 
+
+  **Action:**
+
+  No action required.
+
+### ZWES0061I
+
+  TLS settings: keyring '%s', label '%s', password '%s', stash '%s'
+
+  **Reason:**
+
+  ZSS uses TLS settings: keyring `<keyring>` or `<p12-file>`, label `<cert-label>`, password `"****"` or `(no password)`, stash `<stash-file>` or `(no stash)`.
+
+  **Action:**
+
+  No action required.
+### ZWES0063I
+
+  Caching Service settings: gateway host '%s', port %d
+
+  **Reason:**
+
+  Caching Service settings are gateway host `<Gateway-host>`, port `<Gateway-port>`.
+
+  **Action:**
+
+  No action required.
+
+### ZWES0064I
+
+  Caching Service not configured
+
+  **Reason:**
+
+  Caching Service not configured.
+
+  **Action:**
+
+  No action required.
+### ZWES1100I
+
+  Product Registration is enabled.
+
+  **Reason:**
+
+  Product Registration is enabled.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1101I
+
+  Product Registration is disabled.
+
+  **Reason:**
+
+  Product Registration is disabled.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1102I
+
+  Product Registration successful.
+
+  **Reason:**
+
+  Product Registration successful.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1600I
+
+  JWT will be configured using JWK URL '%s'
+
+  **Reason:**
+
+  JWT will be configured using JSON Web Key(JWK) at URL <url>.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1601I
+
+  Server is ready to accept JWT `with`(or `without`) fallback to legacy tokens
+
+  **Reason:**
+
+  Server is ready to accept JWT `with` or `without` fallback to legacy tokens.
+
+  **Action:**
+
+  No action required.
+
+## ZSS error messages
+
+
+### ZWES1006E
+
+  Error while parsing plugin definition file '%s': '%s'.
+
+  **Reason:**
+
+  An error occurred while parsing `<plugin-definition-file>`: `<error-details>`.
+  
+
+  **Action:**
+
+  Check `<error-details>` and fix the error by editing `<plugin-definition-file>`.
```

Comment:
It's unlikely a non-developer can/should edit a plugindef because it might reside in a read-only location.
They may need to send the error back to the company that wrote the plugin.